style.css

```diff
@@ -0,0 +1,78 @@
+*{
+	text-decoration: none;
+	font-family: 'nunito sans';
+	color: black;
+}
+img{
+	width: 200px;
+	height: 200px;
+}
+body{
+	max-width: 1200px;
+	margin: 0 auto;
+}
+.pink{
+	color: #B67DFF;
+}
+/*-------------- 1- HEADER ----------------*/
+nav{
+	font-size: 18px;
+}
+ul,li,#logo{
+	display: inline-block;
+}
+ul{
+	float: right;
+}
+li{
+	padding-left: 70px;
+}
+span{
+	padding: 5px;
```

Comment:
width and height can be set as 50px to get the circle shape as well as adding padding